restfulapi/src/model/Employee.java

```diff
@@ -0,0 +1,52 @@
+package model;
+
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.xml.bind.annotation.XmlRootElement;
+
+@XmlRootElement
+@Entity(name="Employee")
+public class Employee {
+
+	@GeneratedValue(strategy=GenerationType.AUTO)
+	private int id;
+
+	private String firstName;
+
+	private String lastName;  
+
+	private Category category;
+
+	public int getId() {  
```

Comment:
Please consider use Lombok to avoid create setters and getters for each property in the classes. this only as a comment because we are using lombok in the backend project.